scripts/variantstore/wdl/GvsBulkIngestGenomes.wdl

```diff
@@ -0,0 +1,118 @@
+        # End GvsPrepareBulkImport
+
+        # Begin GvsAssignIds  ## TODO what should the behavior be if we have run this once already and several samples did not load properly?
+        String dataset_name
+        String bq_project_id
+        String call_set_identifier
+
+        File? gatk_override
+        # End GvsAssignIds
+
+        # Begin GvsImportGenomes
+        File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
+
+        # set to "NONE" to ingest all the reference data into GVS for VDS (instead of VCF) output
+        String drop_state = "NONE"
+
+        # The larger the `load_data_batch_size` the greater the probability of preemptions and non-retryable
+        # BigQuery errors so if specifying this adjust preemptible and maxretries accordingly. Or just take the defaults,
+        # those should work fine in most cases.
+        Int? load_data_batch_size
+        Int? load_data_preemptible_override
+        Int? load_data_maxretries_override
+        # End GvsImportGenomes
+
+        ## TODO do we want to add Alt Allele Table creation?
+    }
+
+
+    call GetWorkspaceId
+
+
+    call PrepareBulkImport.GvsPrepareBulkImport as PrepareBulkImport {
+        input:
+            project_id = terra_project_id,
+            workspace_name = workspace_name,
+            workspace_bucket = "fc-" + GetWorkspaceId.workspace_id,
+            samples_table_name = samples_table_name,
+            sample_id_column_name = sample_id_column_name,
+            vcf_files_column_name = vcf_files_column_name,
+            vcf_index_files_column_name = vcf_index_files_column_name
+    }
+
+    call AssignIds.GvsAssignIds as AssignIds {
+        input:
+            ## go = PrepareBulkImport.done, ## TODO we're gonna want to add this, right?
+            dataset_name = dataset_name,
+            project_id = bq_project_id,
+            external_sample_names = read_lines(PrepareBulkImport.sampleFOFN),
+            samples_are_controls = false ## TODO this shouldn't always be false tho
+    }
+
+    call ImportGenomes.GvsImportGenomes as ImportGenomes {
+        input:
+            go = AssignIds.done,
+            dataset_name = dataset_name,
+            project_id = bq_project_id,
+
+            external_sample_names = read_lines(PrepareBulkImport.sampleFOFN),
+            input_vcfs = read_lines(PrepareBulkImport.vcfFOFN),
+            input_vcf_indexes = read_lines(PrepareBulkImport.vcfIndexFOFN),
+
+            ## TODO the following will be kept short term for testing
+            ## is_rate_limited_beta_customer = true,
+            ## beta_customer_max_scatter = 25,
+
+            interval_list = interval_list,
+
+            # The larger the `load_data_batch_size` the greater the probability of preemptions and non-retryable
+            # BigQuery errors so if specifying this adjust preemptible and maxretries accordingly. Or just take the defaults,
```

Comment:
nit
```suggestion
            # BigQuery errors so, if specifying this adjust, preemptible and maxretries accordingly. Or just take the defaults,
```

Reply:
uh that suggestion doesn't look right... 😕